app/models/user.rb

```diff
@@ -7,4 +7,10 @@ class User < ActiveRecord::Base
                     uniqueness: { case_sensitive: false }
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
+
+  def User.digest(string)
```

Comment:
__[rubocop]__
* Use `self.digest` instead of `User.digest`. [:ref](https://github.com/bbatsov/ruby-style-guide#def-self-class-methods)